modules/atom/pom.xml

```diff
@@ -206,7 +206,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-core</artifactId>
+            <artifactId>activemq-broker</artifactId>
```

Comment:
It is just a change coming forced by activemq 5.8: http://activemq.apache.org/58-migration-guide.html

Now its divided on broker, broker-persistence-* and client. However -all is still containing everything. Assuming AFAIK we recommended -all so far. There is no migration guide needed. Anyhow I'll put it just in case.